### ISSUE TITLE
комнаты

### ISSUE THREAD

**Issue by apankov** [Clarid/chat#1]
я не увидел поддержки "комнат" - механизма для уединенной беседы разными группами участников. мы, кажется, обсуждали этот момент

**Comment by Clarid** [comment]
Да, обсуждали. Эту проблему легко решает Faye. Я пробовал использовать его, но возникла проблема подключения этого модуля. Я его подключал, но функции модуля не распознавались. Сейчас изучаю эту проблему. Такая же история была с passport. Авторизацию пришлось писать собственную.

**Comment by apankov** [comment]
any updates?
